app/main.ipynb

```diff
@@ -25,18 +25,22 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 2,
-   "outputs": [],
    "source": [
-    "# write your code here"
+    "import numpy\n",
+    "import numpy as np\n",
+    "import sys\n",
+    "\n",
+    "from numpy import dtype"
```

Comment:
The import `from numpy import dtype` is unnecessary unless you specifically need `dtype` separately. Consider removing it if not used.